chapters/ch10.md

```diff
@@ -1,23 +1,25 @@
 # Having a dev set and metric speeds up iterations
 ->
-
+# Xây dựng một tập phát triển và một phép đo sẽ thúc đẩy nhanh quá trình làm việc.
 It is very difficult to know in advance what approach will work best for a new problem. Even experienced machine learning researchers will usually try out many dozens of ideas before they discover something satisfactory. When building a machine learning system, I will often:
 ->
-
+Thật sự rất khó để biết trước hướng đi nào sẽ phù hợp cho một vấn đề mới. Kể cả những nghiên cứu sinh Machine Learning nhiều kinh nghiệm cũng thường thử nghiệm một vài ý tưởng trước khi có thể khám phá ra một cái gì đó thỏa đáng. Khi xây dựng một hệ thống Machine Learnig, tôi sẽ thường: 
 1. Start off with some ​**idea​** on how to build the system.
 ->
-
+Bắt đầu với một số ý tưởng nào đó về cách xây dựng hệ thống đó.
 2. Implement the idea in ​**code​**.
 ->
-
+Hiện thực hóa ý tưởng đó dưới dạng code.
 3. Carry out an ​**experiment​** which tells me how well the idea worked. (Usually my first few ideas don’t work!) Based on these learnings, go back to generate more ideas, and keep on iterating.
 ->
+Tiến hành một thí nghiệm cho tôi biết rõ rằng ý tưởng đó khả thi. (Thường thì một số ý tưởng đầu tiên sẽ không khả thi!) Dựa vào những kết quả này, quay lại để xây dựng thêm nhiều ý tưởng hơn, và tiếp tục làm điều này lặp đi lặp lại.
```

Comment:
- Thí nghiệm cần được in đậm. 
< Tiến hành **thí nghiệm** giúp kiểm tra tính khả thi của ý tưởng. 
- Đổi vị trí, bỏ "sẽ" 
< Những ý tưởng đầu tiên thường không khả thi!

- Câu này không có chủ ngữ. 
 > Dựa vào những kết quả này, quay lại để xây dựng thêm nhiều ý tưởng hơn, và tiếp tục làm điều này lặp đi lặp lại.
--> gợi ý
"Từ những kết quả đó, chúng ta quay lại thử nghiệm thêm những ý tưởng mới và cứ thế lặp lại.